api/debug.go

```diff
@@ -484,6 +480,6 @@ func isDefaultCallTracer(config *tracers.TraceConfig) bool {
 		return false
 	}
 
-	tracerConfig := json.RawMessage(replayer.TracerConfig)
-	return slices.Equal(config.TracerConfig, tracerConfig)
+	trimmedConfig := strings.ReplaceAll(string(config.TracerConfig), " ", "")
```

Comment:
Can we add comments why we need this?

Reply:
Good point, I'll include some comments in a follow-up PR.